packages/components/src/Lettera/ArticleSchema.purs

```diff
@@ -41,6 +48,13 @@ articleToJsonLd article =
             }
         )
         article.authors
+  , isAccessibleForFree: if article.premium then "False" else "True"
+  , hasPart: if article.premium
+             then Just { "@type": "WebPageElement"
+                       , "isAccessibleForFree": "False"
+                       , "cssSelector": ".premium-only"
```

Comment:
This selector matches with no content on Mosaico. My reading of Google's instructions say that the selector has to match with the part of the document that's paywalled.

Reply:
https://git.ksfmedia.fi/taco/mosaico/-/merge_requests/233